subgraphs/osmosis/src/common/initializer.ts

```diff
@@ -0,0 +1,414 @@
+import { Protobuf } from "as-proto";
+import { BigDecimal, BigInt, cosmos, log } from "@graphprotocol/graph-ts";
+import {
+  Account,
+  Token,
+  _TokenPrice,

```

Comment:
Fix linting error.

Reply:
Fix all linting errors at 6903f52ce28319ef61b2eb05dafdabf73a47568c and 8783d83503c6683bd6261a2189b6986a8745b7a7